### DISCUSSION TITLE
"Conqueror URL", do you like the name?

### DISCUSSION THREAD

**Discussion by MasterInQuestion** [curl/curl#9206] [General]
&#xA0;&#xA0;&#xA0;&#xA0;The proposed explanation to &#x60;curl&#x60;.

**Comment by bagder** [comment]
I don't. It doesn't really make sense.

**Reply by MasterInQuestion** [reply]
[ Quote &#x60;curl&#x60; FAQ # What is cURL? (@ CE 2022-07-26):
[https://curl.se/docs/faq.html#What_is_cURL](https://curl.se/docs/faq.html#What_is_cURL)
    cURL is the name of the project. The name is a play on 'Client for URLs', originally with URL spelled in uppercase to make it obvious it deals with URLs. The fact it can also be read as 'see URL' also helped, it works as an abbreviation for "Client URL Request Library" or why not the recursive version: "curl URL Request Library". ]

    Hardly.

**Comment by MasterInQuestion** [comment]
"connect URL", it likely simply means.
    Inadvertently find that I use cURL as frequently as I use Internet...

    Partly related:
    [ https://github.com/curl/curl/discussions/17930#discussioncomment-14122540](https://github.com/curl/curl/discussions/17930#discussioncomment-14122540)
    (stories of cURL 10,000 days)
